src/github-repo-event-notifier.coffee

```diff
@@ -32,12 +32,16 @@
 #   patcon
 #   parkr
 
+_ = require('lodash')
 inspect = (require('util')).inspect
 url = require('url')
 querystring = require('querystring')
-eventActions = require('./event-actions/all')
 eventTypesRaw = process.env['HUBOT_GITHUB_EVENT_NOTIFIER_TYPES']
 eventTypes = []
+customActions = {}
+try
+  customActions = require.main.require('../../../scripts/github/custom-actions')
```

Comment:
@parkr this a **relative path** to the user's hubot repo, therefore if the repo is called `my-hubot` then this particular file (`github-repo-event-notifier.coffee`) would be located under:

```
my-hubot\node_modules\hubot-github-repo-event-notifier\src
```

Hence the path.

Nevertheless it would be possible to refactor it using [built-in "path" api](https://nodejs.org/api/path.html), and let the user specify the last `dir/file` part (`scripts/github/custom-actions` in this case)

My only question is, where should the user put that configuration?

Cheers!